AdventOfCode.Library/Day13/Part2.fs

```diff
@@ -0,0 +1,21 @@
+module AdventOfCode.Library.Day13.Part2
+
+open AdventOfCode.Library.Day13.Part1
+
+let divider1 = List [ List [ Integer 2 ] ]
+
+let divider2 = List [ List [ Integer 6 ] ]
+
+let solve (input: string) : string =
+    input.TrimEnd().Split "\n\n"
+    |> Seq.collect (fun pair -> pair.Split "\n" |> Seq.map (stringToPacket >> Option.get))
+    |> Seq.append (seq [ divider1; divider2 ])
+    |> Seq.sortWith (fun l r ->
+        match comparePackets l r with
+        | LT -> -1
+        | EQ -> 0
+        | GT -> 1)
```

Comment:
Implementing a proper comparison function for part 1 made this relatively easy. I do wish that F# had a built in `Ordering` type like Haskell though. It's a bit weird to use integers instead.